.clang-format

```diff
@@ -0,0 +1,57 @@
+---
```

Comment:
seems as if this was includeded in error, it is generated automatically from a the other template, as it can vary between versions of clang-format what options are supported or not, and we can programatically generate compatible ones for all versions

Reply:
Might be, just copied it from node's CI. How should I generate a correct one?

Reply:
It's part of the update-clang-format script. Just omit .clang-format and likely add it to .gitignore so it doesn't slip back in